redfish-core/lib/task.hpp

```diff
@@ -99,7 +99,7 @@ struct TaskData : std::enable_shared_from_this<TaskData>
         matchStr(matchIn), index(idx),
         startTime(std::chrono::system_clock::to_time_t(
             std::chrono::system_clock::now())),
-        status("OK"), state("Running"), messages(nlohmann::json::array()),
+        status("OK"), state("Starting"), messages(nlohmann::json::array()),
```

Comment:
This part is also in upstream.
So, would you upstream this PR would and add its Changed-Id here?

Reply:
Starting all the tasks with "Starting" state is not upstreamed.
Also, this change is made just for ibm-specific usecase. If upstreammed, Ed will ask about the usecase -- which will be "resource dump creation" which is downstream only change in bmcweb.

So this stays downstream.